tests/test_s3/test_s3_multipart.py

```diff
@@ -10,7 +10,7 @@
 from moto.s3.responses import DEFAULT_REGION_NAME
 
 
-from moto import settings, mock_s3
+from moto import settings, mock_s3, mock_kms
```

Comment:
This is no longer necessary - I'm assuming that's why the linter fails

Reply:
Thanks for catching this.